source/draw/gpu/opengl/OpenGLPacked.ooc

```diff
@@ -84,7 +85,7 @@ OpenGLPacked: abstract class extends GpuImage {
 					gpuMap add("texture0", image)
 			}
 		}
-		gpuMap useProgram(this, projection * view * model, textureTransform)
+		gpuMap useProgram(this, flat * projection * view * model, textureTransform)
```

Comment:
That is why we use normalized coordinates from -1 to +1 for axis tilt coefficients.